src/OSPSuite.Infrastructure/packages.config

```diff
@@ -5,11 +5,24 @@
   <package id="Dapper" version="1.50.5" targetFramework="net462" />
   <package id="FluentNHibernate" version="2.0.3.0" targetFramework="net462" />
   <package id="Iesi.Collections" version="4.0.2" targetFramework="net462" />
+  <package id="Microsoft.Extensions.Configuration" version="2.2.0" targetFramework="net462" />
+  <package id="Microsoft.Extensions.Configuration.Abstractions" version="2.2.0" targetFramework="net462" />
+  <package id="Microsoft.Extensions.Configuration.Binder" version="2.2.0" targetFramework="net462" />
+  <package id="Microsoft.Extensions.DependencyInjection.Abstractions" version="2.2.0" targetFramework="net462" />
+  <package id="Microsoft.Extensions.Logging" version="2.2.0" targetFramework="net462" />
+  <package id="Microsoft.Extensions.Logging.Abstractions" version="2.2.0" targetFramework="net462" />
+  <package id="Microsoft.Extensions.Options" version="2.2.0" targetFramework="net462" />
+  <package id="Microsoft.Extensions.Primitives" version="2.2.0" targetFramework="net462" />
   <package id="Newtonsoft.Json" version="12.0.1" targetFramework="net462" />
   <package id="NHibernate" version="4.1.1.4000" targetFramework="net462" />
   <package id="OSPSuite.SmartXLS" version="2.6.9.3" targetFramework="net462" />
   <package id="OSPSuite.TeXReporting" version="2.1.1.9" targetFramework="net462" />
   <package id="OSPSuite.Utility" version="3.0.0.3" targetFramework="net462" />
   <package id="SharpZipLib" version="1.1.0" targetFramework="net462" />
+  <package id="System.Buffers" version="4.4.0" targetFramework="net462" />
+  <package id="System.ComponentModel.Annotations" version="4.5.0" targetFramework="net462" />
   <package id="System.Data.SQLite.Core" version="1.0.109.2" targetFramework="net462" />
+  <package id="System.Memory" version="4.5.1" targetFramework="net462" />
```

Comment:
I hate all those dependencies added as sub dependencies. Nothing we can do however